db/bigtable.go

```diff
@@ -1542,279 +1831,400 @@ func (bigtable *Bigtable) SaveValidatorIncomeDetails(epoch uint64, rewards map[u
 	return nil
 }
 
-func (bigtable *Bigtable) GetEpochIncomeHistoryDescending(startEpoch uint64, endEpoch uint64) (*itypes.ValidatorEpochIncome, error) {
-	ctx, cancel := context.WithDeadline(context.Background(), time.Now().Add(time.Second*30))
-	defer cancel()
+func (bigtable *Bigtable) MigrateEpochSchemaV1ToV2(epoch uint64) error {
```

Comment:
**Question:** `SYNC_COMMITTEES_PARTICIPATION_FAMILY` is not touched by the migration file. Is that intended?

Reply:
yes, it is currently not implemented